src/marqo/tensor_search/tensor_search.py

```diff
@@ -329,20 +352,29 @@ def add_documents(config: Config, index_name: str, docs: List[dict], auto_refres
                     # TODO put the logic for getting field parameters into a function and add per field options
                     image_method = index_info.index_settings[NsField.index_defaults][NsField.image_preprocessing][NsField.patch_method]
                     # the chunk_image contains the no-op logic as of now - method = None will be a no-op
-                    content_chunks, text_chunks = image_processor.chunk_image(field_content, 
-                                    device=selected_device,
-                                    method=image_method)            
+                    content_chunks, text_chunks = image_processor.chunk_image(
```

Comment:
the error will also need to be caught here. if an image is chunked then there will be an attempt to retrieve it here prior to the vectorise call

Reply:
add the try except block around chunking and tested

Reply:
Thank you!